tools/wake-format/emitter.cpp

```diff
@@ -1102,13 +1137,33 @@ wcl::doc Emitter::walk_prim(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_publish(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
+  assert(node.id() == CST_PUBLISH);
+
+  MEMO_RET(fmt()
+               .token(TOKEN_KW_PUBLISH)
+               .ws()
+               .walk(WALK_NODE)  // identifier
```

Comment:
I wonder if we want a WALK_ID to be a bit more pedantic than just WALK_NODE?

Reply:
Yeah, we had something like that previously but removed it when we said "every dispatch needs to go through walk_node" 

I'd like to find that change and "revert" it now that we have DISPATCH